### PR TITLE
Retrieve All complaints done

### PR DESCRIPTION
**Pull request by Naijabas**

### CODE REVIEW

File: app/Http/Controllers/ComplaintController.php [hunk 1]
```diff
@@ -142,28 +143,24 @@ public function update(Request $request, $id)
...
-         // return view('backend.transactions.index')->with('error', "Unable to connect to server");
-     }
-      // $client = new \GuzzleHttp\Client();
-
-      // $response = $request->send();
-
-      // dd($response);
+        $host = env('API_URL', 'https://dev.api.customerpay.me/');
+        $url = $host."complaint/delete/5efc5cb9aea77164da871ee9";
+        $request = new HttpRequest();
```
Comment (Jeremiahiro) [inline app/Http/Controllers/ComplaintController.php:148]: I cannot accept this delete route, this is not how to pass :id

File: app/Http/Controllers/ComplaintController.php [hunk 2]
```diff
@@ -142,28 +143,24 @@ public function update(Request $request, $id)
...
+        $request->setUrl($url);
+        $request->setMethod(HTTP_METH_DELETE);
+        
+        $request->setHeaders(array(
+          'postman-token' => 'b6c9656d-4feb-d9f0-2b1d-0d26753ebbcd',
+          'cache-control' => 'no-cache',
```
Comment (Jeremiahiro) [inline app/Http/Controllers/ComplaintController.php:154]: what's all of these?? what has postman-token got to do with anything on this project

File: resources/views/backend/complaints/index.blade.php [hunk 3]
```diff
@@ -37,30 +36,27 @@
...
-                                    <td>{{isset( $response->createdAt) ? $response->createdAt : "2020 07 25"}}</td>
-                                    <td><a href="{{ route('complaint.destroy',$response->_id) }}" class="btn btn-danger " id="deletecomplaint" data-id="{{ $response->_id }}">
-   Delete
-</a>
+                                    <td>{{isset( $response->date) ? $response->date : "2020 07 25"}}</td>
+                                    <td><form action="{{ route('complaint.destroy', $response->_id) }}" method="POST">
+                                            <input type="hidden" name="_method" value="DELETE">
+                                            <input type="hidden" name="_token" value="{{ csrf_token() }}">
+                                        <button class="btn btn-danger">Delete User</button>
+                        </form></td>
```
Comment (Jeremiahiro) [inline resources/views/backend/complaints/index.blade.php:53]: what's with the token here and why am I seeing Delete User??

**Comment by Jeremiahiro** [comment]
Please review your  work again

File: app/Http/Controllers/ComplaintController.php (hunk 1 above)
Comment (Naijabas) [reply app/Http/Controllers/ComplaintController.php:148]: ohh, sorry 
I was testing for delete and perhaps, I'm submitting another thing

File: resources/views/backend/complaints/index.blade.php (hunk 3 above)
Comment (Naijabas) [reply resources/views/backend/complaints/index.blade.php:53]: just a test bro
and the hidden token field is just to send the id to the controller

**Pull request closed without merging**